### PR TITLE
Enable mean operator for a ngraph engine

### PR DESCRIPTION
**Pull request by mozga-intel**
Introduces the nGraph's implementation of the mean operator.

### CODE REVIEW

**Comment by mozga-intel** [comment]
@luotao1, @tensor-tang Can you please review and comment this pull-request?

File: paddle/fluid/operators/ngraph/ops/mean_op.h [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+#include "ngraph/ngraph.hpp"
+#include "paddle/fluid/operators/ngraph/ops/elementwise_scalar_op.h"
+#include "paddle/fluid/platform/ngraph_helper.h"
+
+namespace paddle {
+namespace operators {
+namespace ngraphs {
+
+static void BuildMeanNode(
```
Comment (tensor-tang) [inline paddle/fluid/operators/ngraph/ops/mean_op.h]: why static

Comment (mozga-intel) [reply paddle/fluid/operators/ngraph/ops/mean_op.h]: @tensor-tang done.

**Comment by tensor-tang** [comment]
Please pass the PR_CI_python35

**Comment by mozga-intel** [comment]
@tensor-tang Done.

**Review by luotao1 — APPROVED** [review]
LGTM

**Pull request merged**